### ISSUE TITLE
Did something in the dependency chain break?

### ISSUE THREAD

**Issue by Jrayp** [rebassjs/rebass#510]
NOTE: I am quite new to webdev and may be doing something stupid. Not even sure if my title makes sense.

I deployed my app to heroku, and was getting a bunch of errors suddenly, after changes that "shouldnt" have led to this (I believe)

`TypeError: Cannot use 'in' operator to search for '75em' in 20`

```
  29 |   return paths.join('.').split('.').reduce(function (a, b) {
> 30 |     return a && b in a ? a[b] : null;
  31 |   }, obj);
  32 | };
  33 | export var themeGet = function themeGet(paths, fallback) {
```

In: `../node_modules/styled-system/dist/index.esm.js:30`

Strangely I could not replicate the issue on my local dev version. However, after uninstalling styled-components, and rebass, and reinstalling the packages, I DO get this error, and can not resolve it.

I'm (quite possibly incorrectly) assuming that something in the chain broke, since I didn't get the issue until reinstalling the same version of rebass that i have been using all along. (I believe). 

Could of course also be related to styled-components, I have been using the beta for that, which updates frequently. (Probably shouldnt be using the beta)

tldr: reinstalling the rebass and styled components packages has led to the above error, and I'm a stumped noob.

edit: I have rolled back all the way to styled-components 4.0.0-beta.2 and I still have the problem, so I can only guess that the issue is related to rebass or styled-system. (Or me doing something very clueless of course)

**Comment by Jrayp** [comment]
I installed styled-system 3.1.6 (the one before the recent updates), and things seem to be working locally.
So maybe I was actually right? :P

**Comment by jxnblk** [comment]
Thanks for reporting! I suspect the change in jxnblk/styled-system#297 is the culprit – are you using styled-system directly anywhere in your app? Could also be helpful to see the theme object and some examples of how you're using Rebass components

**Comment by Jrayp** [comment]
There are a few instances of me using styled-system to use the responsive css values:

```js
import { height } from "styled-system";

export const FlexWithHeight = styled(Flex)`
	${height};
`;
```

But I am not sure if that's the issue since the error seems to be related to my usage of font sizes. I also get over 250 of these errors in my app, and I'm not using styled-system (directly) that often.

My theme object looks like this:
```js
export const Theme = {
	breakpoints: ["481px", "769px", "1601px", "2001px"],
	colors: {
		lightShade1: "#FAFAFB",
		lightShade2: "#f2f1f2",
		darkShade1: "#011627",
		darkShade2: "#678",

		main: "#FF9F1C",
		accent1: "#4B9EC1",
		accent2: "#7FA232",

		text1: "#262B4A",
		text2: "#676b80",

		lightShadow: "#CCC",
		darkShadow: "#345"
	},
	fonts: {
		primary: "Open Sans, sans-serif",
		secondary: "PT Sans, Sans-Serif",
		title: "PT Sans, Sans-Serif",
		subtitle: "Open Sans, sans-serif",
		heading: "PT Sans, Sans-Serif",
		label: "Open Sans, sans-serif",
		button: "Open Sans, sans-serif",

		quote: "Alegreya, serif"
	},
	fontSizes: [12, 14, 16, 20, 24, 34, 48, 60, 96, 124, 192],
	shadows: {
		text: "0.05em 0.05em 0.02em rgba(0, 0, 0, .2)"
	}
};
```
And I (quite frankly) still haven't figured out a one-system-fits-all for how to use rebass components, so I'm all over the place with that.

**Comment by jxnblk** [comment]
Going to close this out for now until I have time to dig into the `in` operator issue in styled-system. Ensure you have styled-system >=3.1.11 installed to avoid this issue if you're still seeing it
